bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/obs/ObsListView.scala

```diff
@@ -322,7 +322,7 @@ class ObsListView(shellAdvisor:ShellAdvisor, band:Band) extends BorderPanel with
             case n  => Proposal.targets.set(p0, ts.updated(n, newT))
           }
         }
-
+        
```

Comment:
Why the empty spaces?

Reply:
I have no idea... I guess that's what my editor wanted. (I'm using IntelliJ IDEA Pro.)
Looks like I have invoked the wrath of Travis, so in the process of fixing that, I'll remove the spaces as well.